### PR TITLE
Fix unable to select text on animation bug (#1053)

### LINKED ISSUE #1053: Unable to select text in `How does it work?` animation section
In the text panels explaining each part of the `How does it work?` animation, each text panel is positioned/stacked on top of each other and is only hidden by `opacity: 0;`. So you're unable to select any text on any of the panels except the last one.

https://matrix.org/

## Potential solutions

We should update it to actually hide the panels from the document flow `display: none;` or at least have `pointer-events: none` for hidden panels.

### PR DESCRIPTION
**Pull request by tawandamoyo**
Fixes #1053 , used the suggestion from @MadLittleMods

### CODE REVIEW

File: gatsby/static/js/how-it-works.js [hunk 1]
```diff
@@ -483,14 +483,14 @@ function nextStage() {
 
     var dissolveTime = 500;
 
-    d3.select("#legend" + stageIndex).style("opacity", 1)
+    d3.select("#legend" + stageIndex).style("display", "block")
         .transition()
         .duration(dissolveTime)
-        .style("opacity", 1e-6);
-    d3.select("#legend" + (stageIndex + 1)).style("opacity", 1e-6)
+        .style("display", "none");
```
Comment (MadLittleMods) [inline gatsby/static/js/how-it-works.js:489]: The opacity transition is lost here. We can still have the best of both worlds though.

Or if you can't figure it out, we can always use `pointer-events: none;` on hidden panels so you can click through them.

Comment (tawandamoyo) [reply gatsby/static/js/how-it-works.js]: HI @MadLittleMods , thanks for the feedback. I have used the second option of `pointer-events: none`. If you could kindly review, thanks.

File: gatsby/static/css/how-it-works.css [hunk 2]
```diff
@@ -13,8 +13,9 @@
 .legend {
     position: absolute;
     padding-top: 20px;
-    opacity: 0;
     max-width: 570px;
+    opacity: 0;
+    pointer-events: none;
```
Comment (MadLittleMods) [inline gatsby/static/css/how-it-works.css:18]: Looks good to me 👍

https://user-images.githubusercontent.com/558581/167490575-a9279ffb-6bec-412b-a494-414c094344df.mp4

**Review by MadLittleMods — CHANGES_REQUESTED** [review]
Remove changes to `gatsby/package-lock.json`

File: gatsby/static/css/how-it-works.css [hunk 3]
```diff
@@ -13,8 +13,9 @@
 .legend {
```
[the remark this answers is not in the document]
Comment (tawandamoyo) [reply gatsby/static/css/how-it-works.css:13]: Oh thanks, done.

**Comment by MadLittleMods** [comment]
Thanks for the contribution and fix @tawandamoyo! Now the demo is even more slick 🦦

**Comment by tawandamoyo** [comment]
Thanks for the reviews and suggestions @MadLittleMods , I learnt a lot. And happy to contribute in any way to this revolutionary project!

**Pull request merged**